auto_route_generator/pubspec.yaml

```diff
@@ -3,6 +3,8 @@ description: AutoRoute is a declartive routing solution, where everything needed
 version: 3.1.0
 homepage: https://github.com/Milad-Akarie/auto_route_library
 
+publish_to: none
```

Comment:
Same as above. Just to keep track of it. 🙂

Reply:
Ditto